streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -72,7 +72,8 @@
         this.eosEnabled = StreamsConfig.EXACTLY_ONCE.equals(config.getString(StreamsConfig.PROCESSING_GUARANTEE_CONFIG));
         this.stateDirectory = stateDirectory;
 
-        this.logPrefix = String.format("%s [%s] ", isStandby ? "standby-task" : "task", id);
+        final String threadIdPrefix = String.format("stream-thread [%s] ", Thread.currentThread().getName());
+        this.logPrefix = threadIdPrefix + String.format("%s [%s] ", isStandby ? "standby-task" : "task", id);
```

Comment:
@mjsax @guozhangwang Just added this to the PR, what do you think about including the thread id in the prefix (not just the task)? I can split this into a separate PR if you want to just get this one merged, there may be some other places/classes where this would be useful as well.

Reply:
SGTM to add it in this PR.